SlowCheetah.VisualStudio/Utilities/NugetHandler/NugetPackageHandlerBase.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.ComponentModelHost;
+using Microsoft.VisualStudio.Shell;
+using NuGet.VisualStudio;
+
+namespace SlowCheetah.VisualStudio
+{
+    public abstract class NugetPackageHandlerBase : INugetPackageHandler
+    {
+        protected IServiceProvider package;
+        protected const string pkgName = "SlowCheetah";
+
+        protected NugetPackageHandlerBase(IServiceProvider package)
+        {
+            this.package = package;
+        }
+
+        public bool IsSlowCheetahInstalled(EnvDTE.Project project)
+        {
+            var componentModel = (IComponentModel)Package.GetGlobalService(typeof(SComponentModel));
+            IVsPackageInstallerServices installerServices = componentModel.GetService<IVsPackageInstallerServices>();
```

Comment:
Need to check if anything <= 2.5.15 is installed. You can do this by calling `IVsPackageInstallerServices.GetInstalledPackages` and finding if any match `SlowCheetah` and that version range.